posthog/migrations/0272_alter_organization_plugins_access_level.py

```diff
@@ -0,0 +1,20 @@
+# Generated by Django 3.2.15 on 2022-10-18 06:42
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0271_delete_promptsequencestate"),
+    ]
+
+    operations = [
+        migrations.AlterField(
```

Comment:
would be interesting to look at the table DDL... My guess is this is a no-op for the DB and only helps Django. Nice to add that as a comment to migrations so we know whether to care about migration performance

Reply:
I think you're right. It isn't actually modifying anything (my local DB has no default)